test/all_close_f.cpp

```diff
@@ -368,7 +355,7 @@ INSTANTIATE_TEST_CASE_P(
 
 // Test the exact bounds near +0.f
 //
-// With mantissa_bits = 8, tolerance_bits = 2
+// With tolerance_bits = 18
```

Comment:
You should probably explain why 18 bits for bfloat is the same as 2 bits for float.